pnet-idp-client-openidconnect/src/main/java/at/porscheinformatik/idp/openidconnect/PartnerNetOAuth2AuthorizationRequestResolver.java

```diff
@@ -39,6 +40,16 @@ public static UriComponentsBuilder requestNistAuthenticationLevels(UriComponents
         return uri;
     }
 
+    public static UriComponentsBuilder requestPreselectTenant(UriComponentsBuilder uri,
+    String preseletedTenant)
```

Comment:
Sorry for the late reply. There is a typo in the paramter name. It should be presele **c** tedTenant